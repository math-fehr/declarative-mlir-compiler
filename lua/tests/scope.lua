local a = 0

if true then
  local b = 1
  print(a)
  print(b)
end

print(a)
print(b)

for i=1,1 do
  print(a)
  print(i)
end

print(a)
print(i)

if true then
  local a = 1
  print(a)
end
print(a)
