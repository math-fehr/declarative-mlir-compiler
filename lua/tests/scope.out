0       
1       
0       
nil     
0       
1       
0       
nil     
1       
0       
